packages/cosmic-swingset/lib/ag-solo/vats/repl.js

```diff
@@ -1,6 +1,8 @@
 import { makeEvaluators } from '@agoric/evaluate';
 import harden from '@agoric/harden';
 
+const swingSetSymbol = Symbol.for('SwingSet');
```

Comment:
I'm not sure why, but I'm a tiny bit uncomfortable about expanding the contract between liveSlots, and the javascript environment it hosts. There a little bit of magic here that we'll have to include in our definition of Jessie or "the swingset javascript environment" or whatever we end up calling the userspace of a vat.

But I don't think that's a reason to not do this.

Reply:
Jessie doesn't have symbols, BTW.  And if the contract is broken, all that happens is Presences aren't easily detectable by the "swingset javascript environment".

Reply:
Jessie not having symbols seems like a problem with this approach. So far, everything passable is a subset of what can be represented and manipulated from Jessie.

Reply:
The symbol is not passable or passed.  It's between the liveSlots interface and user code, all of which lives in the same vat.  So, the "detect if it's a presence" code cannot be written within Jessie, but I think she need not concern herself with it.

Reply:
Maybe we need `HandledPromise.isPresence(p)` ? If we can agree on that, then there's no need for the special symbol usage (the REPL can use `isPresence`), and this PR reduces to just ensuring presences have no ownProperties, as originally desired.

Reply:
@erights and I have agreed upon #804.  I will be implementing the first stage of it in this PR.